test/unittests/CandidateTest.php

```diff
@@ -29,14 +29,14 @@ class CandidateTest extends TestCase
      */
     private $_candidateInfo
         = array(
-           'RegistrationCenterID'     => '2',
+           'RegistrationCenterID'     => 2,
```

Comment:
If this supposed to simulate the data returned by the database? If so, I think it should remain a string. The database returns everything as a string (or null) regardless of the MySQL type.